doc/scoped_enum.qbk

```diff
@@ -46,7 +46,7 @@ The user can portably declare such enumeration as follows:
     }
     BOOST_SCOPED_ENUM_DECLARE_END(future_errc)
 
-These macros allows to use `future_errc` in almost all the cases as an scoped enum.
+These macros allow using `future_errc` in almost all the cases as a scoped enum.
```

Comment:
"Allow to use" seems correct to me. Is it not? I'm not a native speaker.

Reply:
Please see https://english.stackexchange.com/q/60271.